shared/output/cloud/speech_v1/test/google/cloud/speech/v1/speech_test.rb

```diff
@@ -255,6 +255,7 @@ def test_streaming_recognize
         request.to_a.each do |r|
           assert_kind_of ::Google::Cloud::Speech::V1::StreamingRecognizeRequest, r
           assert_equal Gapic::Protobuf.coerce({}, to: ::Google::Cloud::Speech::V1::StreamingRecognitionConfig), r.streaming_config
+          assert_equal :streaming_config, r.streaming_request
```

Comment:
This seems wrong. Google::Cloud::Speech::V1::StreamingRecognizeRequest has a oneof of two fields: `streaming_config` and `audio_content`. In a stream, the first message is supposed to contain a `streaming_config`, but all subsequent messages should contain `audio_content`. This suggests that we are asserting that _all_ messages contain `streaming_config`.

Reply:
nvm, looks like the test data always uses the first choice.

Reply:
we're not creating a message with `audio_content` anywhere, our testing generates all `streaming_config` so while it is counterintuitive it's not incorrect